app/const.py

```diff
@@ -42,7 +42,7 @@ class MIMETYPE(StrEnum):
     ),
     "carlisle.gov.uk": (
         ("Cumberland Council",),
-        ("Carlisle City Centre",),
+        ("Workington", "Cleator Moor", "Millom", "Carlisle", "Carlisle City Centre", "Maryport Town Centre"),
```

Comment:
Do we also need to delete these from the current e-mail address they're mapped to, or are we expecting two different gov.uk e-mails to be able to submit for these places?

Reply:
I believe they should both be able to submit for these places - @gidsg are you able to confirm this?

Reply:
@Joel-Pearce I think people from the LAs may be using either depending what their AD account is mapped to. So I'd suggest keeping both.

Reply:
In that case I've approved.